### PR TITLE
[API Compatibility] Add C Sink API expand_as, logsumexp, argmax, argmin, all

### PR DESCRIPTION
**Pull request by hushenwei2000**
### PR Docs
| API | PR |
| :------- | :------: | 
| torch.Tensor.expand_as  | https://github.com/PaddlePaddle/Paddle/pull/74882 | 
| torch.(Tensor.)logsumexp  | https://github.com/PaddlePaddle/Paddle/pull/74859 | 
| torch.(Tensor.)argmax  | https://github.com/PaddlePaddle/Paddle/pull/74856  | 
| torch.(Tensor.)argmin  | https://github.com/PaddlePaddle/Paddle/pull/74856  | 
| torch.(Tensor.)all | https://github.com/PaddlePaddle/Paddle/pull/74691  | 

### PR APIs

```bash
"torch.Tensor.expand_as",
"torch.logsumexp",
"torch.Tensor.logsumexp",
"torch.argmax",
"torch.Tensor.argmax",
"torch.argmin",
"torch.Tensor.argmin",
"torch.all",
"torch.Tensor.all",
```

### CODE REVIEW

File: docs/api/paddle/all_cn.rst [hunk 1]
```diff
@@ -7,13 +7,22 @@ all
...
 参数
 :::::::::
     - **x** （Tensor）- 输入变量为多维 Tensor，数据类型为 bool、float32、float64、int32、int64。
+      别名： ``input``
     - **axis** （int | list | tuple，可选）- 计算逻辑与运算的维度。如果为 None，则计算所有元素的逻辑与并返回包含单个元素的 Tensor 变量，否则必须在 :math:`[−rank(x),rank(x)]` 范围内。如果 :math:`axis [i] <0`，则维度将变为 :math:`rank+axis[i]`，默认值为 None。
+      别名： ``dim``
     - **keepdim** (bool，可选) - 是否在输出 Tensor 中保留减小的维度。除非 keepdim 为 True，否则输出 Tensor 的维度将比输入 Tensor 小一维，默认值为 False。
     - **name** (str，可选) - 具体用法请参见 :ref:`api_guide_Name`，一般无需设置，默认值为 None。
 
+关键字参数
```
Comment (sunzhongkai588) [inline docs/api/paddle/all_cn.rst]: 在上面的函数签名里 `.. py:function:: paddle.all(x, axis=None, keepdim=False, name=None)` 是不是也对应加一下？不然有误解

File: docs/api/paddle/logsumexp_cn.rst [hunk 2]
```diff
@@ -10,13 +10,22 @@ logsumexp
...
 参数
 ::::::::::
     - **x** (Tensor) - 输入的 Tensor，数据类型为：float16，float32、float64，维度不超过 4 。
+      别名： ``input``
     - **axis** (int|list|tuple，可选) - 指定对 ``x`` 进行计算的轴。``axis`` 可以是 int、list(int)、tuple(int)。如果 ``axis`` 包含多个维度，则沿着 ``axis`` 中的所有轴进行计算。``axis`` 或者其中的元素值应该在范围[-D, D)内，D 是 ``x`` 的维度。如果 ``axis`` 或者其中的元素值小于 0，则等价于 :math:`axis + D`。如果 ``axis`` 是 None，则对 ``x`` 的全部元素计算 logsumexp。默认值为 None。
+      别名： ``dim``
     - **keepdim** (bool，可选) - 是否在输出 Tensor 中保留减小的维度。如果 ``keepdim`` 为 True，则输出 Tensor 和 ``x`` 具有相同的维度(减少的维度除外，减少的维度的大小为 1)。否则，输出 Tensor 的形状会在 ``axis`` 上进行 squeeze 操作。默认值为 False。
     - **name** (str，可选) - 具体用法请参见 :ref:`api_guide_Name`，一般无需设置，默认值为 None。
 
+关键字参数
```
Comment (sunzhongkai588) [inline docs/api/paddle/logsumexp_cn.rst]: 同理，在函数签名里加一下

File: docs/api/paddle/all_cn.rst (hunk 1 above)
Comment (hushenwei2000) [reply docs/api/paddle/all_cn.rst]: 已修改

File: docs/api/paddle/logsumexp_cn.rst (hunk 2 above)
Comment (hushenwei2000) [reply docs/api/paddle/logsumexp_cn.rst]: 已修改

**Review by sunzhongkai588 — APPROVED** [review]
LGTM

**Review by sunzhongkai588 — APPROVED** [review]
LGTM
